src/style.scss

```diff
@@ -48,6 +48,10 @@
   }
 }
 
+auro-menuoption {
+  pointer-events: none;
```

Comment:
removing pointer events has weird side effects. This is to remove the hover event of adding the element into the span, right? Could we just over-write the hover UI?

Reply:
What are you actually concerned about here? The auro-menuoption in the trigger should *never* have pointer events. What can you call out that will actually go wrong here?

Reply:
Removing pointer-events only eliminates visual cues; the functionality of pointer-events is disabled. However, the element remains present and selectable with alternative input methods.

While I can't produce a specific use-case, there are a11y risks with this as well. 

My recommendation is to rewrite the hover UI versus using something potentially problematic as pointer-events.